src/server/db/models/GameModel.ts

```diff
@@ -16,12 +21,11 @@ const GameSchema = new Schema<IGame>({
   name: { type: String, required: true, unique: true },
   themes: {
     type: [Schema.Types.ObjectId],
-    ref: "Theme",
+    default: [],
```

Comment:
Keep the ref to the `Theme` document, this is necessary for other backend routes